File: PackXprezApp/src/app/packXprez-interfaces/ScheduledPickup.ts
export interface IScheduledPickup {

  id: number;
  emailId: string
 pickupPinCode: number;
 deliveryPinCode:number
 shipmentType:string
 packageLen:number
 packageBreadth:number
 packageHeight:number
 packageWeight:number
packingReq:number
 deliveryOpt: string
 timeSlot:string
 pickupAddress: string
 deliveryAddress: string
insurance: number
 amount: number
 payment: string
  awbNumber: number
  name: string
}

File: Infosys.PackXprez.WebService/Models/ScheduledPickup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Infosys.PackXprez.WebService.Models
{
    public class ScheduledPickup
    {
        public int Id { get; set; }
        public string EmailId { get; set; }
        public decimal PickupPinCode { get; set; }
        public decimal DeliveryPinCode { get; set; }
        public string ShipmentType { get; set; }
        public decimal PackageLen { get; set; }
        public decimal PackageBreadth { get; set; }
        public decimal PackageHeight { get; set; }
        public decimal PackageWeight { get; set; }
        public short PackingReq { get; set; }
        public string DeliveryOpt { get; set; }
        public DateTime TimeSlot { get; set; }
        public string PickupAddress { get; set; }
        public string DeliveryAddress { get; set; }
        public short Insurance { get; set; }
        public decimal Amount { get; set; }
        public string Payment { get; set; }
        public long? AwbNumber { get; set; }
        public string Name { get; set; }
    }
}
